docs/api-testing/environments.md

```diff
@@ -7,51 +7,63 @@ description: "Gain a wide range of options to mix and match your test settings w
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-Our Environment feature provides you with a range of options to mix and match your Test settings.
+Our Environments feature provides you with a range of options to mix and match your Test settings.
+
+If properly set up, any API Testing test can be run against any environment. This can be achieved by turning the URL and any other part into variables that can be overwritten using the environments.  This allows you, for example, to set the default location as input and override those amounts with the environments feature without actually changing the Test.
+
+The **Environments** section lets you change anything, not just environments. You can run the Test against a certain environment, using a different API key and data source.
 
 
 ## What You'll Need
 * An existing API Testing Project and Test. For details on how to create them, see [API Testing Quickstart](/api-testing/quickstart/).
 * Recommended: review [Creating Reusable Variables and Snippets with the Vault](/api-testing/vault/)
 
-## Using Environments
 
-Once a domain is parametrized, you can override a variable, if needed.
+## Creating Environments
 
-To access and create new environments:
+Once a domain is parametrized, you can override a variable if needed.
```

Comment:
Hi @LoryApiFortress , I pulled this from the legacy doc, which talked about creating a variable for the mastiff.apifortress.com domain. But I'm not sure if it's needed anymore.

Reply:
@kimsauce it's just an example how to parametrise a domain and creating variables for each part. You can leave it as is or replace it with any other domain